Code/GraphMol/AddHs.cpp

```diff
@@ -73,7 +73,7 @@ void setHydrogenCoords(ROMol *mol, unsigned int hydIdx, unsigned int heavyIdx) {
       // --------------------------------------------------------------------------
       dirVect.z = 1;
       // loop over the conformations and set the coordinates
-      for (ROMol::ConformerIterator cfi = mol->beginConformers();
+      for (auto cfi = mol->beginConformers();
```

Comment:
Once this pull request is finished, we need to figure out how to enable this idiom:

```
for (auto &conf : mol->getConformers())
  ...
```

Shouldn't too hard to marshall the beginConformers and endConformers() points into a struct that exposes a begin and end method.